src/Gradient.php

```diff
@@ -372,13 +375,9 @@ function Gradient($x, $y, $w, $h, $type, $stops = [], $colorspace = 'RGB', $coor
 				// All values are set in parseMozGradient - so won't appear here
 				$coords = [0, 0, 1, 0]; // default for original linear gradient (L2R)
 			}
-			$s = ' q';
-			$s .= sprintf(' %.3F %.3F %.3F %.3F re W n', $x * Mpdf::SCALE, ($this->mpdf->h - $y) * Mpdf::SCALE, $w * Mpdf::SCALE, -$h * Mpdf::SCALE) . "\n";
-			$s .= sprintf(' %.3F 0 0 %.3F %.3F %.3F cm', $usew * Mpdf::SCALE, $useh * Mpdf::SCALE, $usex * Mpdf::SCALE, ($this->mpdf->h - ($usey + $useh)) * Mpdf::SCALE) . "\n";
 		} // RADIAL
-		else if ($type == 3) {
+		else if ($type == self::TYPE_RADIAL) {
```

Comment:
Could we have `elseif` everywhere and on the same line as previous closing brace? With the constant used for type, the comment is unnecessary.